pkg/manager/member/tikv_scaler.go

```diff
@@ -243,6 +245,12 @@ func (s *tikvScaler) preCheckUpStores(tc *v1alpha1.TidbCluster, podName string)
 	pdClient := controller.GetPDClient(s.deps.PDControl, tc)
 	// get the number of stores whose state is up
 	upNumber := 0
+
+	if !tc.TiKVBootStrapped() {
+		klog.Infof("TiKV of Cluster %s/%s is not bootstrapped yet, skip pre check when scale in TiKV", tc.Namespace, tc.Name)
+		return true, nil
+	}
+
```

Comment:
Move to the beginning of this function or checking this in the caller function?

Reply:
ok